File: src/main/java/Entites/Transport.java
package Entites;


import java.time.LocalDateTime;
public class Transport {
    private int id;
    private String type;
    private String capacite;
    private Float tarif;
    private String depart;
    private String arrivee;
    private LocalDateTime datedepart;
    public Transport() {}
    public Transport(String type, String capacite, Float tarif, String depart, String arrivee,LocalDateTime datedepart) {
        this.type = type;
        this.capacite = capacite;
        this.tarif = tarif;
        this.depart = depart;
        this.arrivee = arrivee;
        this.datedepart = datedepart;

    }
    public Transport(int id, String type, String capacite, Float tarif, String depart, String arrivee,LocalDateTime datedepart) {
        this.id = id;
        this.type = type;
        this.capacite = capacite;
        this.tarif = tarif;
        this.depart = depart;
        this.arrivee = arrivee;
        this.datedepart = datedepart;

    }

    public String getType() {
        return type;
    }

    public void setType(String type) {
        this.type = type;
    }

    public int getId() {
        return id;
    }

    public void setId(int id) {
        this.id = id;
    }

    public String getCapacite() {
        return capacite;
    }

    public void setCapacite(String capacite) {
        this.capacite = capacite;
    }

    public Float getTarif() {
        return tarif;
    }

    public void setTarif(Float tarif) {
        this.tarif = tarif;
    }
    public String getDepart() {
        return depart;
    }

    public void setDepart(String depart) {
        this.depart = depart;
    }
    public String getArrivee() {return arrivee;}

    public void setArrivee(String arrivee) {
        this.arrivee = arrivee;
    }
    public LocalDateTime getDateDepart() {return datedepart;}

    public void setDateDepart(LocalDateTime datedepart) {
        this.datedepart = datedepart;
    }





    @Override
    public String toString() {
        return "Reservation{" +
                "user=" + type +
                ", id=" + id +
                ", tarif='" + tarif + '\'' +
                ", Experience='" + capacite + '\'' +
                ", départ=" + depart +'\'' +
                ", arrivee=" + arrivee +'\'' +
                ", date départ=" + datedepart +
                '}';
    }
}

File: src/Entity/Transport.php
<?php

namespace App\Entity;

use App\Repository\TransportRepository;
use Doctrine\ORM\Mapping as ORM;

#[ORM\Entity(repositoryClass: TransportRepository::class)]
class Transport
{
    #[ORM\Id]
    #[ORM\GeneratedValue]
    #[ORM\Column(type: 'integer')]
    private ?int $id = null;

    #[ORM\Column(type: 'string', length: 250)]
    private string $type = '';

    #[ORM\Column(type: 'string', length: 250)]
    private string $capacite = '';

    #[ORM\Column(type: 'float')]
    private float $tarif = 0.0;

    #[ORM\Column(type: 'string', length: 250)]
    private string $depart = '';

    #[ORM\Column(type: 'string', length: 250)]
    private string $arrivee = '';

    #[ORM\Column(type: 'datetime')]
    private ?\DateTimeInterface $datedepart = null;

    #[ORM\ManyToOne(targetEntity: Utilisateur::class)]
    #[ORM\JoinColumn(name: 'user_id', referencedColumnName: 'id', nullable: true)]
    private ?Utilisateur $user_id = null;

    public function getId(): ?int
    {
        return $this->id;
    }

    public function setId(int $value): self
    {
        $this->id = $value;

        return $this;
    }

    public function getType(): string
    {
        return $this->type;
    }

    public function setType(string $value): self
    {
        $this->type = $value;

        return $this;
    }

    public function getCapacite(): string
    {
        return $this->capacite;
    }

    public function setCapacite(string $value): self
    {
        $this->capacite = $value;

        return $this;
    }

    public function getTarif(): float
    {
        return $this->tarif;
    }

    public function setTarif(float $value): self
    {
        $this->tarif = $value;

        return $this;
    }

    public function getDepart(): string
    {
        return $this->depart;
    }

    public function setDepart(string $value): self
    {
        $this->depart = $value;

        return $this;
    }

    public function getArrivee(): string
    {
        return $this->arrivee;
    }

    public function setArrivee(string $value): self
    {
        $this->arrivee = $value;

        return $this;
    }

    public function getDatedepart(): ?\DateTimeInterface
    {
        return $this->datedepart;
    }

    public function setDatedepart(\DateTimeInterface $value): self
    {
        $this->datedepart = $value;

        return $this;
    }

    public function getUser_id(): ?Utilisateur
    {
        return $this->user_id;
    }

    public function setUser_id(?Utilisateur $user): self
    {
        $this->user_id = $user;

        return $this;
    }
}